src/extension-client/remix-extension.ts

```diff
@@ -1,5 +1,9 @@
 import { Message, Api, PluginRequest } from '../types'
 
+export interface RemixExtensionOptions {
+  useCustomBootStrapTheme: false
```

Comment:
Define the type of `useCustomBootStrapTheme` as a `boolean`. Default value should be managed by an external method or in the constructor.